src/components/Counter.js

```diff
@@ -20,20 +35,19 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => {/* Fill me in */ }}>
+                <button onClick={() => this.props.increment(this.props.count) }>
```

Comment:
`increment` function does not take any variable, so passing `this.props.count` does not do anything here.